src/vs/workbench/parts/files/electron-browser/dirtyFilesTracker.ts

```diff
@@ -66,6 +68,7 @@ export class DirtyFilesTracker implements IWorkbenchContribution {
 	}
 
 	private onUntitledDidChangeDirty(resource: URI): void {
+
```

Comment:
I prefer to not add empty lines in the beginning of a method, please stick to the coding format used elsewhere in the files.